bedrock/firefox/templates/firefox/new/wait-face/base.html

```diff
@@ -0,0 +1,105 @@
+
+{% add_lang_files "firefox/new/wait-face" %}
+
+{% extends "firefox/base-pebbles.html" %}
+
+{% block extra_meta %}
+  <meta name="robots" content="noindex,follow">
+{% endblock %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{_('Free Web Browser for Android, iOS, Desktop | Firefox')}}{% endblock %}
+{% block page_desc %}{{_('Millions of people around the world trust Firefox Web browsers on Android, iOS and desktop computers. Fast. Private. Download now!')}}{% endblock %}
+
+{% block page_css %}{% endblock %}
+
+{% block body_id %}firefox{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block string_data %}
+  data-pixels="{% for pixel in settings.TRACKING_PIXELS %}{{ pixel }}{% if not loop.last %}::{% endif %}{% endfor %}"
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+  <main role="main">
+  <section class="intro content-section">
+    <div class="content">
+      <div class="copy">
+        <img class="logo" src="{{ static('img/logos/firefox/logo-quantum.png') }}" width="100" height="100" alt="" />
+        {% block head_content %}{% endblock %}
+        {% block primary_download_button %}{% endblock %}
+      </div>
+    </div>
+  </section>
+
+  {% block details %}
+  <section class="details content-section">
+    <div class="content">
+      <ul>
+        <li>
+        {% block content_1 %}
+          <h3>{{ _('Runs 2x faster') }}</h3>
+          <p>
+          {% trans url='https://blog.mozilla.org/firefox-de/firefox-quantum-superschnell-und-schont-den-speicher/' if LANG == 'de' else 'https://blog.mozilla.org/firefox/quantum-performance-test/' %}
+            The New Firefox has a new engine. <a href="{{ url }}">Twice as fast</a>? Yes. Crazy-powerful engine?
+            Yes. Less time waiting for browser pages to load? Double yes, yes. Get the best Firefox yet.
+          {% endtrans %}
+          </p>
+        {% endblock %}
+        </li>
+        <li>
+        {% block content_2 %}
+          <h3>{{ _('Uses 30% less memory') }}</h3>
+          <p>
+          {% trans url='https://blog.mozilla.org/firefox-de/firefox-quantum-superschnell-und-schont-den-speicher/' if LANG == 'de' else 'https://blog.mozilla.org/firefox/quantum-performance-test/' %}
+            <a href="{{ url }}">Lighter than Chrome</a> means more of your computer resources are freed up and run smoother.
+            You can thank the New Firefox multiprocess architecture for that. Your computer will thank you back.
+          {% endtrans %}
+          </p>
+        {% endblock %}
+        </li>
+        <li>
+        {% block content_3 %}
+          <h3>{{ _('Stands for good') }}</h3>
+          <p>
+          {% trans %}
+            Did you know Firefox works with the not-for-profit Mozilla? We keep the Internet healthier by supporting
+            programs like tech education for girls, truth in online journalism, and promoting civility in comments
```

Comment:
We need to change "tech education for girls" to "tech education for women". @peiying2 is aware of this change, and will comment here on how we should handle it in this template.

(FWIW, the German translation does not appear to need updating, as it reads "Für die digitale Bildung von Frauen." - "For the digital education of women.")